lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1504,7 +1504,7 @@ LowererMD::LowerTry(IR::Instr * tryInstr, IR::JnHelperMethod helperMethod)
     if (tryInstr->m_opcode == Js::OpCode::TryCatch || this->m_func->DoOptimizeTry())
     {
         // Arg 6 : hasBailedOutOffset
-        IR::Opnd * hasBailedOutOffset = IR::IntConstOpnd::New(this->m_func->m_hasBailedOutSym->m_offset, TyInt32, this->m_func);
+        IR::Opnd * hasBailedOutOffset = IR::IntConstOpnd::New(this->m_func->m_hasBailedOutSym->m_offset + tryInstr->m_func->GetInlineeArgumentStackSize(), TyInt32, this->m_func);
```

Comment:
>+ tryInstr->m_func->GetInlineeArgumentStackSize() [](start = 104, length = 49)

Why we don't do the same thing in arm32?

Reply:
We need to do the same thing on arm32, apparently.